src/app/public/modules/inline-delete/fixtures/inline-delete.component.fixture.html

```diff
@@ -6,21 +6,24 @@
 </button>
 <div
   id="inline-delete-fixture"
+  [@.disabled]="true"
 >
-  <button
+  <button *ngIf="showCoveredButtons"
```

Comment:
I'm hiding these buttons during the a11y checks because they were failing the color-contrast rule (the overlay was appearing over the buttons; as I understand it, this failure isn't a big deal because the contents shouldn't be discernible when the inline-delete is covering them).